src/recipes/apertass.json

```diff
@@ -24,7 +24,7 @@
         "Pour a small bottle of Tassoni Soda cedar and add a tip of Aperol, completing with the Soda.",
         "Stir gently. Garnish with a slice of orange."
     ],
-    "image": "apertass.jpg>",
+    "image": "apertass600x400.jpg>",
```

Comment:
This should still be `apertass.jpg` and the image filename should also be `apertass.jpg`

Reply:
Done